src/webclients/components/sdfv/transformation/transformation.ts

```diff
@@ -346,6 +373,8 @@ export function showTransformationDetails(xform: JsonTransformation): void {
         text: xform.docstring,
     }).appendTo(xformInfoContainer);
 
+    // TODO: Silence error messages that are being printed if this doesn't
+    // exist.
```

Comment:
can we do this easily?

Reply:
I tried, but so far I haven't found a reliable way of doing it since the error is logged to the console deep inside an external Ajax call :/